templates/footer.html

```diff
@@ -66,6 +66,7 @@ <h3 class="h4 text-white">Company</h3>
                  href="https://github.com/simple-login/app/projects/1">Roadmap</a></li>
           <li><a class="list-group-item text-white footer-item"
                  href="https://simplelogin.io/contact/">Contact Us</a></li>
+            <li><a class="list-group-item text-white footer-item" href="/dashboard/support">Support</a></li>
```

Comment:
I think the support link can live in the header, for example 
<img width="930" alt="image" src="https://user-images.githubusercontent.com/3167780/153208951-2d91a3e2-afa0-4cbd-9852-c1422026a128.png">

Reply:
This is something I wanted to ask you. Perfect. I'll change into that.